src/tools/server/sdk/adGuard/adGuard.schema.ts

```diff
@@ -32,7 +32,7 @@ export const adGuardApiStatusResponseSchema = z.object({
 export const adGuardApiFilteringStatusSchema = z.object({
   filters: z.array(
     z.object({
-      url: z.string().url(),
+      url: z.string(),
```

Comment:
Consider to use https://zod.dev/?id=strings in combination with https://github.com/validatorjs/validator.js